### PR TITLE
[FIX] mail: restore style of messaging menu icon

### PR DESCRIPTION
**Pull request by seb-odoo**
- restore "solid" icon
- restore muted style when no notification
- opportunity is taken to fix off-pixel of activity menu

task-2278416

### CODE REVIEW

**Comment by alexkuhn** [comment]
branch is red

File: addons/mail/static/src/scss/variables.scss [hunk 1]
```diff
@@ -10,3 +10,7 @@ $o-mail-attachment-image-size: 100px !default;
 $o-mail-sidebar-icon-opacity: 0.7 !default;
 $o-mail-chat-sidebar-width: 250px !default;
 $o-mail-partner-avatar-size: 24px !default;
+
+@mixin o-mail-systray-no-notification-style{
```
Comment (alexkuhn) [inline addons/mail/static/src/scss/variables.scss:14]: space

File: addons/mail/static/src/scss/systray.scss [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
         > i {
             font-size: larger;
         }
     }
     &.o_no_notification > a {
-        opacity: 0.5;
-        > i {
-            transform: translateY(0px);
-        }
+        @include o-mail-systray-no-notification-style();
```
Comment (alexkuhn) [inline addons/mail/static/src/scss/systray.scss:9]: I assume this is for activity menu, correct?

File: addons/mail/static/src/components/messaging_menu/messaging_menu.xml [hunk 3]
```diff
@@ -5,8 +5,8 @@
         <li class="o_MessagingMenu" t-att-class="{ show: messagingMenu.isOpen, 'o-mobile': env.messaging.device.isMobile }">
             <!-- TODO FIXME restore mobile stuff here (activeMobileThread) -->
             <t t-if="!env.messaging.device.isMobile or true">
-                <a class="o_MessagingMenu_toggler o-no-caret" t-att-class="{ show: messagingMenu.isOpen }" href="#" t-on-click="_onClickToggler" data-toggle="dropdown" data-display="static" title="Conversations" role="button" aria-expanded="false">
-                    <i class="o_MessagingMenu_icon fa fa-comments-o" role="img" aria-label="Messages"/>
+                <a class="o_MessagingMenu_toggler o-no-caret" t-att-class="{ show: messagingMenu.isOpen, o_MessagingMenu_togglerNoNotification: !messagingMenu.counter }" href="#" t-on-click="_onClickToggler" data-toggle="dropdown" data-display="static" title="Conversations" role="button" aria-expanded="false">
```
Comment (alexkuhn) [inline addons/mail/static/src/components/messaging_menu/messaging_menu.xml:8]: newly introduced classname looks like a modifier, therefore I would expect it to follow our naming convention for modifiers (`o-<modifierName>` or `o_<ComponentName>_<partName>-<modifierName>`).

**Comment by seb-odoo** [comment]
test adapted

File: addons/mail/static/src/scss/variables.scss (hunk 1 above)
Comment (seb-odoo) [reply addons/mail/static/src/scss/variables.scss:14]: ok

File: addons/mail/static/src/scss/systray.scss (hunk 2 above)
Comment (seb-odoo) [reply addons/mail/static/src/scss/systray.scss:9]: yes, old code... might as well mixin it already in prevision

File: addons/mail/static/src/components/messaging_menu/messaging_menu.xml (hunk 3 above)
Comment (seb-odoo) [reply addons/mail/static/src/components/messaging_menu/messaging_menu.xml:8]: `o-noNotification`?

Comment (alexkuhn) [reply addons/mail/static/src/components/messaging_menu/messaging_menu.xml:8]: `o-noNotification` or `o-no-notification`, both are good (in fact we seem to have used `<modifier-name>`, which is ok with shorter version but not so longer one).

So let's take `o-no-notification` to be consistent right now.

Comment (seb-odoo) [reply addons/mail/static/src/components/messaging_menu/messaging_menu.xml]: ok

**Pull request closed without merging**